neuronpp/utils/simulation.py

```diff
@@ -152,7 +166,7 @@ def run(self, runtime, stepsize=None, delay_between_steps=1, debug=False):
             * Simulation stepsize
         """
         if not self.warmup_done:
-            self.reset()
+            self.init_simulation()
```

Comment:
Use either reset or reinit

Reply:
reinit() is fine, but reset() may indicates that you remove objects as well, which is not true.